app/js/results/smaaResultsController.js

```diff
@@ -41,6 +52,43 @@ define(['clipboard', 'require'], function(Clipboard) {
         $scope.state = SmaaResultsService.addSmaaResults($scope.state);
       });
     }
+
+    function initUncertaintyOptions() {
+      $scope.hasNoStochasticMeasurements = SmaaResultsService.hasNoStochasticMeasurements($scope.aggregateState);
+      $scope.hasNoStochasticWeights = SmaaResultsService.hasNoStochasticWeights($scope.aggregateState);
```

Comment:
variables and functions being identically named is a bit smelly especially when variables can contain functions